### ISSUE TITLE
Pass/Fail is misleading

### ISSUE THREAD

**Issue by joppekroon** [CoreyGinnivan/whocanuse#60]
You apply contrast calculations after applying the simulation and base pass/fail results on the adjusted colours. With this you make it seem like #663399 vs #FF99FF fails WCAG, but this is not the case. 

Contrast calculation should be done on the original colours only. The 3:1, 4.5:1, and 7:1 contrast ratios already take into account if the end result would be sufficiently visible/readable to people with these visual impairments.

**Comment by Kilian** [comment]
Relevant section of the explainer to support the assertion that re-calculating the contrast ratio after transforming the colors is wrong:

> A contrast ratio of 3:1 is the minimum level recommended by [[ISO-9241-3](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#ISO-9241-3)] and [[ANSI-HFES-100-1988](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#ANSI-HFES-100-1988)] for standard text and vision. The 4.5:1 ratio is used in this Success Criterion to account for the loss in contrast that results from moderately low visual acuity, congenital **or acquired color deficiencies**, or the loss of contrast sensitivity that typically accompanies aging.

https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html
